images/win/scripts/Installers/Install-Haskell.ps1

```diff
@@ -33,3 +33,7 @@ Write-Host 'Installing cabal...'
 Choco-Install -PackageName cabal
 
 Invoke-PesterTests -TestFile 'Haskell'
+
+# install minimal ghcup, utilizing pre-installed msys2 at C:\msys64
```

Comment:
nitpick: I think we should split this command.

```
$bootstrapHaskell = Invoke-WebRequest https://www.haskell.org/ghcup/sh/bootstrap-haskell.ps1 -UseBasicParsing
Invoke-Command -ScriptBlock ([ScriptBlock]::Create($bootstrapHaskell)) -ArgumentList $false, $true, $true, $false, $false, $false, $false, C:\, "", C:\msys64, C:\cabal
```

Reply:
Done